UIKitDZ/UIKitDZ/ViewController.swift

```diff
@@ -1,19 +1,160 @@
-//
-//  ViewController.swift
-//  UIKitDZ
-//
-//  Created by Қадыр Маратұлы on 08.02.2024.
-//
+// ViewController.swift
+// Copyright © RoadMap. All rights reserved.
 
 import UIKit
 
+/// ViewController
 class ViewController: UIViewController {
```

Comment:
1. 
```suggestion
final class ViewController: UIViewController {
```
2. Из названия неясно, что делает этот контроллер. Стандартный контроллер проекта тоже можно переименовывать.